### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
### Summary

SUMMARY_GOES_HERE
fixed typo
### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ transformations, and more through plugins.
   updated across nodes.
 - **Scalability**: Distributed by nature, Kong scales horizontally by simply
   adding nodes.
-- **Performance**: Kong handles load with ease by scaling and using NGINX at
+- **Performance**: Kong handles the load with ease by scaling and using NGINX at
```
Comment (levfishbluefish) [inline README.md]: unnecessary change

**Comment by guanlan** [comment]
We are closing it, thanks for the PR @sakshamtaneja21
